Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM openjdk:11-jdk AS build
```

Comment:
Can we add

```
LABEL org.opencontainers.image.source=https://github.com/bfidatadigipres/bfi-iiif-logging
```

So that GitHub ties the built images to the repository?